src/base/bittorrent/session.h

```diff
@@ -582,6 +593,8 @@ namespace BitTorrent
 #if (LIBTORRENT_VERSION_NUM >= 10200)
         void handleAlertsDroppedAlert(const lt::alerts_dropped_alert *p) const;
 #endif
+        void handleStorageMovedAlert(const lt::storage_moved_alert *p);
+        void handleStorageMovedFailedAlert(const lt::storage_moved_failed_alert *p);
```

Comment:
Seems this is a const function.

Reply:
Nope. Non-const method is called within it.